### PR TITLE
[RU] Fixes for 11.2

### PR DESCRIPTION
**Pull request by REALMWTH**
# **SCP:SL Translation PR Template**

## **Language Name**
Russian

## **State Type of change (New language, changed existing Etc.)**
Fixes of some inconsistencies between English and Russian translation.
Also contains rephrased versions of translation for easier understanding.

# **Checklist:**
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

File: Translations/ru/LoadingHints.txt [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
-Micro H.I.D. очень громкий во время зарядки. Используйте его тогда, когда у ваших целей нет возможности убежать.
+Бронебойные патроны (AP) лучше всего действуют против бронированных целей.
+Экспансивные патроны (JHP) наносят больший урон по небронированным целям, но слабее против бронированных.
+Удерживайте клавишу перезарядки, чтобы разрядить оружие и забрать патроны в инвентарь.
+Оружие Повстанцев Хаоса мощное, но совместимые с ним боеприпасы не хранятся в комплексе. Используйте их с умом.
+Вы можете разрушать двери гранатами.
+Граната может быть брошена с меньшей силой, если бросить ее, используя правую кнопку мыши.
+С помощью Micro H.I.D. можно очень быстро убить большинство SCP, но он имеет ограниченный запас энергии. Пользуйтесь с умом!
+Перед использованием, Micro H.I.D. требуется около 5-ти секунд на зарядку.
+Поддерживайте Micro H.I.D. в заряженном положении, удерживая правую клавишу мыши. При этом медленно расходуется его энергия.
```
Comment (GuyFawkes76) [inline Translations/ru/LoadingHints.txt:28]: Второе предложение плохо звучит в сочетании с первым.

File: Translations/ru/LoadingHints.txt [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
+Micro H.I.D. издаёт очень громкий звук во время зарядки. Используйте его тогда, когда у ваших целей нет возможности убежать.
 Зарядка Micro H.I.D. сопровождается устрашающими звуками, которых достаточно, чтобы разрушить планы ваших врагов.
-Бутылка SCP-207 не только увеличивает вашу скорость, но и частично восстанавливает ваше здоровье.
+Употребление содержимого бутылки SCP-207 не только увеличивает вашу скорость, но и частично восстановит ваше здоровье.
```
Comment (GuyFawkes76) [inline Translations/ru/LoadingHints.txt:31]:
```suggestion
Употребление содержимого бутылки SCP-207 не только увеличит вашу скорость, но и частично восстановит ваше здоровье.
```

File: Translations/ru/LoadingHints.txt [hunk 3]
```diff
@@ -1,63 +1,63 @@
...
 Можно выпить до четырёх бутылок SCP-207, что значительно увеличит вашу скорость.
 SCP-500 не только исцеляет вас, но также снимает большинство негативных эффеков, при этом обеспечивая регенерацию.
-Хоть SCP-268 делает вас невидимым для людей, SCP-079 и SCP-939 все еще могут вас заметить.
+Хоть SCP-268 делает вас невидимым для людей, SCP-079 и SCP-939 всё ещё могут вас заметить.
 SCP-268 не спасёт вас от SCP-096, если вы посмотрели на него.
 SCP-018 крайне непредсказуем. Если его бросили - ищите укрытие.
-Если бросить SCP-018 в коридоре, его можно использовать для остановки преследователей.
-SCP-914 может менять любые предметы, включая оружие и боеприпасы.
-SCP-914 может перезаряжать радио и Micro H.I.D.
+Бросив SCP-018 в коридоре, можно остановить ваших преследователей.
```
Comment (GuyFawkes76) [inline Translations/ru/LoadingHints.txt:37]: Спорно. Можно и так, но, мне кажется, можно лучше. Если ничего не придумается, сделаем так.

File: Translations/ru/LoadingHints.txt [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
+SCP-914 может изменять любые предметы, включая оружие и боеприпасы.
+SCP-914 может перезаряжать рации и Micro H.I.D.
 SCP-079 вначале слаб, но становится сильнее в течение матча.
-SCP-079 может блокировать двери, отключать свет, и управлять тесла-воротами.
-Тесла-ворота, на которые смотрит SCP-079, всегда будут активны.
+SCP-079 может блокировать двери, отключать свет, и активировать ворота Тесла.
+Ворота Тесла, за которыми наблюдает SCP-079, всегда будут активны.
```
Comment (GuyFawkes76) [inline Translations/ru/LoadingHints.txt:42]: Тут тоже не знаю насчёт ворот Тесла или Тесла-ворот.

File: Translations/ru/SCP330.txt [hunk 5]
```diff
@@ -3,19 +3,19 @@
...
 Имеет сочный запах винограда. У вас начинают течь слюни.
 Даёт временное сопротивление урону и медленно восстанавливает здоровье.
 Красная конфета
-Сильный аромат вишни наполняет комнату. Это немного... <i>слишком</i> сильно.
+Сильный аромат вишни наполняет комнату. Пахнет немного... <i>слишком</i> сильно.
 Быстро восстанавливает здоровье.
 Зелёная конфета
 От конфеты исходит неприятный травяной запах.
-Даёт длительную регенерацию здоровья и временный иммунитет к большинству статусных эффектов.
+Даёт длительную регенерацию здоровья и временный иммунитет к большинству побочных эффектов.
```
Comment (GuyFawkes76) [inline Translations/ru/SCP330.txt:16]: А они побочные?

File: Translations/ru/Subtitles.txt [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
-На объекте сохраняется серьёзная угроза безопасности -- соблюдайте осторожность.
+В пределах комплекса сохраняется серьёзная угроза безопасности -- соблюдайте осторожность.
 успешно уничтожен. Причина не указана.
 уничтожен SCP-[scp].
 успешно уничтожен Автоматической Системой Охраны.
 успешно уничтожен боеголовкой Альфа.
-успешно сдержан научным персоналом.
-успешно сдержан персоналом класса-Д.
-успешно сдержан Повстанцами Хаоса.
+успешно сдержан Научным Сотрудником.
```
Comment (GuyFawkes76) [inline Translations/ru/Subtitles.txt:9]: А надо ли здесь единственное число? Может, множественное?

File: Translations/ru/Subtitles.txt [hunk 7]
```diff
@@ -1,27 +1,27 @@
...
+успешно сдержан Персоналом Класса D.
+успешно сдержан Повстанцем Хаоса.
```
Comment (GuyFawkes76) [inline Translations/ru/Subtitles.txt:11]: Тот же вопрос про единственное число.

File: Translations/ru/Subtitles.txt [hunk 8]
```diff
@@ -1,27 +1,27 @@
 Отряд Мобильной Опергруппы Epsilon-11, обозначенный как [designation], прибыл в комплекс.<split>Всему персоналу рекомендуется действовать согласно стандартным протоколам эвакуации, пока отряд МОГ не прибудет к месту назначения.<split>
 Ожидается восстановление условий содержания для [count] SCP объектов.
 Ожидается восстановление условий содержания для 1 SCP объекта.
-На объекте сохраняется серьёзная угроза безопасности -- соблюдайте осторожность.
+В пределах комплекса сохраняется серьёзная угроза безопасности -- соблюдайте осторожность.
```
Comment (GuyFawkes76) [inline Translations/ru/Subtitles.txt:4]: Не знаю. "В пределах комплекса" звучит неестественно.

File: Translations/ru/Subtitles.txt [hunk 9]
```diff
@@ -1,27 +1,27 @@
...
 SCP-[scp]
 успешно сдержан. Отряд неизвестен.
 успешно сдержан отрядом [designation].
-утерян в процессе обеззараживания.
+утрачен в процессе обеззараживания.
 [current] из [max] генераторов активировано.
 <split>Все генераторы успешно активированы.
 Перезагрузка через 3... 2... 1...
-Комплекс вернулся в рабочий режим.
+Функционирование комплекса вернулось к штатному режиму.
```
Comment (GuyFawkes76) [inline Translations/ru/Subtitles.txt:19]: В оригинале лучше, но не значит, что хорошо.

File: Translations/ru/Summary.txt [hunk 10]
```diff
@@ -14,4 +14,4 @@
 Выполнение миссии заняло: [summary_round_minutes] минут и [summary_round_seconds] секунд.
 [summary_scp_terminated] Уничтожено Объектов SCP
 Следующий раунд начнётся через: [summary_next_round_countdown] секунд.
-В общей сложности SCP было убито [summary_scpfrags] людей.
\ No newline at end of file
+Убитых Объектами SCP людей: [summary_scpfrags]
```
Comment (GuyFawkes76) [inline Translations/ru/Summary.txt:17]: Зачем?

**Review by GuyFawkes76 — CHANGES_REQUESTED** [review]
Открыт для дискуссий.

**Comment by Rnen** [comment]
Closing this for now as requested changes has not been met. Reopen when changes has been made.

**Pull request closed without merging**